LootLockerServerSDK/Source/LootLockerServerSDK/Public/LootLockerServerForBlueprints.h

```diff
@@ -937,4 +939,177 @@ class LOOTLOCKERSERVERSDK_API ULootLockerServerForBlueprints : public UObject
      */
     UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Purchases", meta = (DeprecatedFunction, DeprecationMessage = "This method has been deprecated. We will soon release an updated purchase system, so please refrain from using this method."))
     static void CheckPurchaseStatusForPlayerByPlatformTransactionID_IncludeProducts(int PlayerID, int PlatformTransactionID, const FLootLockerServerPurchaseStatusWithProductsResponseBP& OnCompletedRequest);
+
+    //==================================================
+    // Player Progressions https://ref.lootlocker.com/server-api/#player-progressions
+    //==================================================
+
+    /**
+     * Get a list of progressions for the specified player
```

Comment:
Maybe add to the description of how many will be returned with this, since we also have the paginated response.